### ISSUE TITLE
Fatal Error Class 'WP_Rocket\Plugin' not found in 3.7

### ISSUE THREAD

**Issue by hebbet** [wp-media/wp-rocket#3032]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version
- Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
i tried the 3.7 release on a test site. i can't active the plugin

**To Reproduce**

1. install the 3.7 release on WordPress 5.5
2. Try to active it

**Expected behavior**
i can active the plugin

**Screenshots**
![image](https://user-images.githubusercontent.com/1423115/91412140-a2cfb780-e849-11ea-9d31-6eb20a1c7284.png)

**Additional context**
```
An error of type E_ERROR was caused in line 44 of the file /var/www/html/wp-content/plugins/wp-rocket/inc/main.php. Error message: Uncaught Error: Class 'WP_Rocket\Plugin' not found in /var/www/html/wp-content/plugins/wp-rocket/inc/main.php:44
Stack trace:
#0 /var/www/html/wp-includes/class-wp-hook.php(287): rocket_init('')
#1 /var/www/html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#2 /var/www/html/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#3 /var/www/html/wp-settings.php(409): do_action('plugins_loaded')
#4 /var/www/html/wp-config.php(29): require_once('/var/www/html/w...')
#5 /var/www/html/wp-load.php(37): require_once('/var/www/html/w...')
#6 /var/www/html/wp-admin/admin.php(34): require_once('/var/www/html/w...')
#7 /var/www/html/wp-admin/options-general.php(10): require_once('/var/www/html/w...')
#8 {main}
  thrown
```

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by arunbasillal** [comment]
@hebbet Thanks for the details. Since we had not officially released 3.7 at the time you opened this issue, I hope you installed it form GitHub? If so, did you install the dependencies using composer? 

We added another commit after this, so either way, please revert back to 3.6.4 and re-download 3.7 again. 

Please let me know how it goes.

**Comment by hellofromtonya** [comment]
@hebbet The files in the repo are not fully ready for transfer to a website. Why? They don't have Composer or dependencies packaged with them. We do this through the Ansible build process to bundle the WP Rocket zip file and make it available through your account or directly in your website's Plugin UI.

When downloading the zip files from this repo, you'd also need to do the following:

1. Unzip it.
2. Open a command line tool (like terminal).
3. Navigate to where the Rocket folder is.
4. Type: `composer install --no-dev --no-scripts` to load Composer.
5. Zip up the folder.

Now it's ready to transfer to your site.

Lots of steps I know. The easier way to do it is to update through the Plugin admin UI, as the package is fully prepared and ready for updating.
